test/recipes/80-test_cmp_http.t

```diff
@@ -277,18 +275,21 @@ sub start_mock_server {
     die "Invalid port: $server_port" unless $server_port =~ m/^\d+$/;
     my $pid = open($server_fh, "$cmd|") or die "Trying to $cmd";
     print "Pid is: $pid\n";
+    my $accept_msg = '';
     if ($server_port eq "0") {
         # Find out the actual server port
         while (<$server_fh>) {
             print;
             s/\R$//;                # Better chomp
             next unless (/^ACCEPT/);
-            $server_port = $server_tls = $kur_port = $pbm_port = $1
-                if m/^ACCEPT\s.*?:(\d+)$/;
+            $accept_msg = $1 if /^ACCEPT\s.*:(\d+)$/;
             last;
         }
     }
-    return $pid if $server_port =~ m/^(\d+)$/;
+    if ($accept_msg =~ m/^(\d+)$/) {
```

Comment:
You can simplify this now to `if ($accept_msg ne '') {`

Reply:
Good point

Reply:
Hmmm, I realised the logic around this isn't quite right, still.  So, uhm, somewhat bigger change...